src/Spout/Writer/Common/Manager/Style/StyleRegistry.php

```diff
@@ -57,6 +57,17 @@ protected function hasStyleAlreadyBeenRegistered(Style $style)
     {
```

Comment:
This method is no longer relevant, as we have the serialized style everywhere we call it.
I'd remove it and only keep your new method.